applications/nexus/nex2ecl.cpp

```diff
@@ -16,37 +16,148 @@
 */
 
 #include <cctype>
-
-#include <string>
-
+#include <getopt.h>
```

Comment:
This should be protected in some way. The symbol `ERT_HAVE_GETOPT` does the cmake check to look for `getopt.h` - my suggestion would be to make the complete build of `nex2ecl` conditional on that symbol.

Reply:
This was forgotten, will add.